README.md

```diff
@@ -0,0 +1,57 @@
+# Source for [rust-for-rustaceans.com](https://rust-for-rustaceans.com)
+
+Home of Reviews and Errata for ["Rust for Rustaceans" by Jon Gjengset](https://nostarch.com/rust-rustaceans).
+
+![book cover art with excited Ferris (Rust mascot)](https://github.com/jonhoo/rust-for-rustaceans.com/blob/main/RustforRustaceans_cover.png?raw=true "Rust for Rustaceans")
+
+# Submit Errata
+
+Found a bug?
+
+Just take a look at [accepted Errata pull requests](https://github.com/jonhoo/rust-for-rustaceans.com/pulls?q=is%3Apr+is%3Aclosed+errata) and submit in similar form.
+
+## Example
+
+Branch / Commit / PR name: "errata print01-ch{nn}-lst{nn} syntax error in example"
+
+````markdown
+---
+chapter: 7
+page: 103
+kind: code
+reporter: Jon Gjengset
+date: 2022-01-09
+---
+
+Listing 7-1 has three closing parens
+
+```rust
+macro_rules! test_battery {
+  ($($t:ty as $name:ident),*)) => {
+    // ...
+  }
+}
+```
+
+but should have only two
+
+```rust
+macro_rules! test_battery {
+  ($($t:ty as $name:ident),*) => {
+    // ...
+  }
+}
+```
+````
+
+Note:
+
+- `reporter` can be any identifier you wish, such as
+  - a name or nickname (ex: Jon Gjengset)
+  - a username or handle (ex: @jonhoo)
+  - perhaps an email, url, etc
+- `kind` may be one of
```

Comment:
Currently `kind` isn't actually interpreted in any way, so any value goes. If I did ever ascribe semantics to them, I'd probably go through and fix up all the existing errata anyway. But having a listing doesn't hurt :+1: